cpp/doxygen/developer_guide/BENCHMARKING.md

```diff
@@ -23,7 +23,7 @@ benchmarks in `cpp/benchmarks/copying/scatter.cu`.
 In the interest of improving compile time, whenever possible, test source files should be `.cpp`
 files because `nvcc` is slower than `gcc` in compiling host code. Note that `thrust::device_vector`
 includes device code, and so must only be used in `.cu` files. `rmm::device_uvector`,
-`rmm::device_buffer` and the various `column_wrapper` types described in [Testing](TESTING.md)
+`rmm::device_buffer` and the various `column_wrapper` types described in [Testing](testing)
```

Comment:
problem with this notation is that while browsing the github repository, these links are broken. is there any workaround for this?

Reply:
Unfortunately I don't see a way to keep all three (Sphinx, doxygen, and Github) views of this working. When we're finally fully happy with the Sphinx docs and disable the doxygen builds altogether, I'm happy to revisit this and see if there's a way to at least keep Sphinx + GH working, but I don't think we should try until we've removed doxygen HTML altogether.